tests/common/test_VanDerCorput.cpp

```diff
@@ -99,16 +99,13 @@ TEST(VanDerCorput, FirstNineIteratorValuesWithEndpoints)
   EXPECT_DOUBLE_EQ(7. / 8, *itr);
 }
 
-TEST(VanDerCorput, ScaleProperlyOverSpan)
-{
-  VanDerCorput vdc{2};
-  EXPECT_DOUBLE_EQ(2.0 / 2, vdc[0].first);
-  EXPECT_DOUBLE_EQ(2.0 / 4, vdc[1].first);
-  EXPECT_DOUBLE_EQ(6.0 / 4, vdc[2].first);
-  EXPECT_DOUBLE_EQ(2.0 / 8, vdc[3].first);
-  EXPECT_DOUBLE_EQ(10. / 8, vdc[4].first);
-  EXPECT_DOUBLE_EQ(6.0 / 8, vdc[5].first);
-  EXPECT_DOUBLE_EQ(14. / 8, vdc[6].first);
```

Comment:
Why not keeping this test? Isn't this still a valid test?

Reply:
I figured the replaced test would suffice for both. I brought this back and added a separate test for this bugfix.